---
title: “Exit Try”只能出现在“Try”语句内
ms.date: 07/20/2015
f1_keywords:
- vbc30393
- bc30393
helpviewer_keywords:
- BC30393
ms.assetid: b8651df3-a32f-478c-a6d8-aa0ef584155f
ms.openlocfilehash: 53c8ee60907db16ac97147100636e413f1cbc15e
ms.sourcegitcommit: 5c1abeec15fbddcc7dbaa729fabc1f1f29f12045
ms.translationtype: MT
ms.contentlocale: zh-CN
ms.lasthandoff: 03/15/2019
ms.locfileid: "58043353"
---
# <a name="exit-try-can-only-appear-inside-a-try-statement"></a>“Exit Try”只能出现在“Try”语句内
`Exit Try` 只能出现在 `Try` 块语句内。 有冗余的 `Exit Try` 语句，或者 `Exit Try` 语句出现在其对应 `Try` 块的边界之外。  
  
 **错误 ID:** BC30393  
  
## <a name="to-correct-this-error"></a>更正此错误  
  
1.  找到并删除不必要的 `Exit Try` 语句。  
  
2.  移动`Exit Try`语句与您的代码中适当的位置。  
  
## <a name="see-also"></a>请参阅

- [Try...Catch...Finally 语句](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
